pkg/processor/runtime/dotnetcore/runtime.go

```diff
@@ -53,10 +53,13 @@ func NewRuntime(parentLogger logger.Logger, configuration *runtime.Configuration
 		return nil, errors.Wrap(err, "Failed to create runtime")
 	}
 
-	// set status to ready
-	newDotnetCoreRuntime.SetStatus(status.Ready)
+	err = newDotnetCoreRuntime.AbstractRuntime.StartRuntime()
```

Comment:
1. It's already called a runtime, so this can simply be `Start`
2. I'm OK with moving the start away from creation, but have a hard time with the fact that this seeps into each runtime (making it slightly more annoying to add runtimes). Can't we simply make workers call start on their runtime after creation? That way it's up to the caller to decide when to start